sql/firefox_desktop_exact_mau28_by_dimensions_v1.sql

```diff
@@ -25,12 +22,11 @@ SELECT
   country,
   distribution_id
 FROM
-  inactive_days
+  clients_last28_v1
 WHERE
   client_id IS NOT NULL
   AND submission_date = @submission_date
 GROUP BY
-  submission_date,
```

Comment:
I like having `submission_date` here because it means you can do a full backfill by simply commenting out `AND submission_date = @submission_date`. This technique also relies on selecting `submission_date` rather than `@submission_date AS submission_date` as changed here.